Purchases/Public/Purchases.swift

```diff
@@ -1098,18 +1098,9 @@ extension Purchases: PurchasesOrchestratorDelegate {
 }
 
 // MARK: Deprecated
-// Deprecated functions extension
-public extension Purchases {
-
-    /**
-     * Enable debug logging. Useful for debugging issues with the lovely team @RevenueCat.
-     */
-    @available(*, deprecated, message: "use Purchases.logLevel instead")
-    @objc static var debugLogsEnabled: Bool {
-        get { logLevel == .debug }
-        set { logLevel = newValue ? .debug : .info }
-    }
 
+// Note: these aren't in `Deprecations.swift` because they're used in tests.
```

Comment:
🤔 
In addition to tests, these can still be used by customers, since they're deprecated but haven't been removed yet. 
Perhaps we could clarify that in comments?

Reply:
Yup, I'll remove the comment.